app/src/main/res/values-h570dp/integers.xml

```diff
@@ -5,10 +5,7 @@
 <resources>
     <!-- Used by the Echo Show 1
```

Comment:
nit: technically, this can be used by Echo Show 2 too so maybe, "Smallest device that uses this file is Echo Show 1, at time of writing."

Alternatively, we're duplicating the much more intuitive description in device_reference so maybe we should remove this and reference that file. It's up to you.